src/hw/actions/transaction.js

```diff
@@ -88,6 +89,10 @@ const reducer = (state: State, e: Event): State => {
       return { ...initialState, transactionSignError };
     }
     case "signed":
+      log(
+        "transaction-summary",
+        `✔️ has been signed! ${JSON.stringify(e.signedOperation)}`
```

Comment:
signed operation is not serializable. We need to pass throuh `toSignedOperationRaw` (indeed it was also probably a mistake in cli.

I suggest we should also put it in 3rd param. maybe like this:

```
log("transation-summary", "✔️ has been signed!", { signedOperation: toSignedOperationRaw(e.signedOperation) })
```